tests/test_instance_incremental.py

```diff
@@ -123,9 +148,49 @@ def test_instance_balanced(balanced_data):
     dataset = InMemoryDataset(*balanced_data)
     scenario = InstanceIncremental(dataset, nb_tasks=5)
 
+    c = 0
+
     for taskset in scenario:
         bincount = np.bincount(taskset._y)
         assert bincount[0] == 5
         assert bincount[1] == 10
         assert bincount[2] == 2
         assert bincount[3] == 3
+
+        c += len(taskset)
+
+    assert c == 100
+
+
+@pytest.fixture
+def imbalanced_data():
+    y = np.concatenate((
+        np.ones((22,), dtype=np.uint32) * 0,
+        np.ones((53,), dtype=np.uint32) * 1,
+        np.ones((19,), dtype=np.uint32) * 2,
+        np.ones((7,), dtype=np.uint32) * 3,
+    ))
+    x = np.ones((len(y), 4, 4, 3), dtype=np.uint8)
+    return x, y
+
+
+def test_instance_imbalanced(imbalanced_data):
```

Comment:
le nom de ce test devrait plutot etre du genre: test_nb_samples_instance_incremental, non ?

Reply:
Not really, because there is also another test doing the same thing but with different data.

But you're right the name is bad, and I've changed them.

Reply:
ok